src/Lobby.ts

```diff
@@ -527,7 +527,7 @@ export class Lobby {
           this.mapId = c.params[0];
           this.mapTitle = c.params[1];
           const changer = this.host ? `by ${c.type === BanchoResponseType.BeatmapChanged ? this.host.name : 'Bot'}` : '';
-          this.logger.info(`Beatmap has been changed ${changer}: https://osu.ppy.sh/b/${this.mapId} ${this.mapTitle}`);
+          this.logger.info(`Beatmap has been changed ${changer}:\n${this.mapTitle}\n<https://osu.ppy.sh/b/${this.mapId}>`);
```

Comment:
`<link>` makes it so that Discord will not make an embed for that specific link.

In console it will be shown as `<linkHere>` as well.